src/scrub_stderr.jl

```diff
@@ -16,7 +16,7 @@ redefinition_regexes =
 """ `scrub_stderr(body::Function, pats::Regex...)` executes `body` without
 outputting any warning that matches one of the `pats`.
 
-Pattern example: r"WARNING\: redefining constant .*\n" """
+Pattern example: r"WARNING\\: redefining constant .*\n" """
```

Comment:
I know very little about regex, but it seems you're right, `WARNING:` would be fine. Could you change them please?